frontend/src/pages/PlaceOrder.jsx

```
@@ -44,71 +43,67 @@ const PlaceOrder = () => {
+            product.size = size;
+            product.quantity = cartItems[productId][size];
+            orderItems.push(product);
           }
         }
       }
-      let orderData = {
+    }
+    return orderItems;
+  };
+  const placeCODOrder = async (orderData, token) => {
+    const res = await axios.post(backendUrl + "/api/order/place", orderData, {
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)